packaging/nnstreamer.spec

```diff
@@ -170,6 +171,7 @@ popd
 %defattr(-,root,root,-)
 %license LICENSE
 %{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_*.so
+%{_prefix}/lib/nnstreamer/decoders/libnnstreamer_decoder_*.so
```

Comment:
please update line 193, exclude decoder_*.so

Reply:
I removed both %exclude because %{_libdir}/*.so does NOT cover %{_prefix/lib/nnstreamer/...